EnviroSense.Web/ViewModels/ApiKeys/CreateDeviceApiKeyViewModel.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace EnviroSense.Web.ViewModels.ApiKeys;
+
+public class CreateDeviceApiKeyViewModel
+{
+    [MinLength(6)]
+    [MaxLength(120)]
+    [Required]
+    public string? Name { get; set; }
```

Comment:
```suggestion
    public string Name { get; set; }
```

because this field is required and validation will not pass if not provided. So, it doesn't make sense to use null identifier